src/api/git/reset.int.test.ts

```diff
@@ -44,7 +44,7 @@ describe('soft', () => {
 
     const directory = await fs.promises.readdir(dir);
 
-    expect(commits[0].oid).toEqual(initialCommit);
+    expect(commits[0]?.oid).toEqual(initialCommit);
```

Comment:
It's unfortunate that this applies to tests too; ideally we'd have an ESLint rule so we could disable it on test files.

It's a bit of a wash in this example, but I'd be open to use of `!` in tests. A runtime error is harmless here and would equally indicate that the test failed.